src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -263,6 +283,7 @@ export async function startTransformByQ() {
             }
         } else {
             transformByQState.setToFailed()
+            resultStatusMessage = 'JobFailed'
             let displayedErrorMessage = CodeWhispererConstants.transformByQFailedMessage
```

Comment:
Can a Job Completion Time out? if yes, perhaps we should cover it's metrics

Reply:
great call! yes we should track that job status